### PR TITLE
adding recommended conferences from kanban board

### PR DESCRIPTION
**Pull request by aayatsali**
Adding list of recommended conferences. Also took a look at the kanban board for inspo and added a few other events type links

### CODE REVIEW

**Review by natalialuzuriaga — APPROVED** [review]
Do we want to add Pycon here? Otherwise, LGTM!

File: growing/recommendedconferences.md
Comment (decause-gov) [inline growing/recommendedconferences.md:6]: This one can be removed before/after merging :)

**Review by decause-gov — APPROVED** [review]
LGTM +1 🚢 🎤

**Pull request merged**
